conftest.py

```diff
@@ -200,21 +200,41 @@ def pytest_collection_modifyitems(items, config):
         if tmp not in selected_services:
             excluded_service.append(tmp)
 
+    # ec2 does not follow the naming conventions, all classes except the following should be run:
+    excluded_ec2_test_cases = [
+        "test_vm_export.py",
+        "test_vm_import.py",
+        "test_utils.py",
+        "test_server.py",
+        "test_reserved_instances.py",
+        "test_monitoring.py",
+        "test_ip_addresses.py",
+        "helpers.py",
+        "test_amazon_dev_pay.py",
+    ]
     # filter tests based on pattern - e.g. every test that includes test_{service_name}
     for item in items:
         for service in selected_services:
             if item in deselected_items or item in selected_items:
                 continue
             test_class_name = item._nodeid.split("::")[0].split("/")[-1]
+            test_package_name = item._nodeid.split("/")[1]
             if any([True for x in excluded_service if x in test_class_name]):
                 deselected_items.append(item)
             elif any([True for x in excluded_test_cases if x in item._nodeid]):
```

Comment:
nit: same as below. I think it's cleaner with `(x in y) for x in xs` otherwise it's always a check of [True, True, ...] vs. [] and actually wouldn't even need the `any` at all